### ISSUE TITLE
Document currently existing community generated videos, documentation, resources related to Kin development on the SOL blockchain

### ISSUE THREAD

**Issue by m4thfr34k** [m4thfr34k/KinCommunityRoadmap#5]

**Comment by squidlingz** [comment]
We need a database for this. I can write great databases in ms access but it doesnt work online which is a shame.

Where and what format did u have in mind?

**Comment by m4thfr34k** [comment]
Can be as simple as an update to a readme.md file that we host in one of the appropriate repos. Some examples are:

- https://github.com/avelino/awesome-go
- https://github.com/awesomedata/awesome-public-datasets
- https://github.com/ramitsurana/awesome-kubernetes

We want this list of resources to be as accessible as possible.

**Comment by m4thfr34k** [comment]
If this is something you'd like to start then let me know and I'll assign the item to you.

**Comment by m4thfr34k** [comment]
I created a new repo, located [here](https://github.com/KinshipFamily/awesome-Kin), and started the template readme that is similar to the other lists I mentioned above.

**Comment by squidlingz** [comment]
yes asign me please.  I'll start slow because i've never used github before.

**Comment by squidlingz** [comment]
I made a branch and made a change and submitted the change, then made a
pull request which I think sends it to you for authorisation to merge it
into the main branch.

If I'm correct, I'll start adding more content.

**Comment by m4thfr34k** [comment]
Reviewed the PR and merged the change. It's perfect. I'll assign this item to you.
